raft.go

```diff
@@ -800,6 +812,9 @@ func (r *raft) appendEntry(es ...pb.Entry) (accepted bool) {
 	for i := range es {
 		es[i].Term = r.Term
 		es[i].Index = li + 1 + uint64(i)
+		if es[i].Type == pb.EntryNormal {
+			traceReplicate(r)
+		}
```

Comment:
Note the proposal may be dropped, see line 820 ~ 827

Reply:
hmmm, let me move the tracing after the drop checking.